package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@plotly/dash-player",
+  "name": "dash-player",
```

Comment:
We need the `@plotly` because `dash-player` is already taken. https://www.npmjs.com/package/dash-player

Reply:
Will be fixed with #48